### ISSUE TITLE
Reduce amount of required technic machines 

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#108]
specing
Idea to reduce amount of technic machines required: 
machine speed/action upgrades.
E.g. stack processing upgrade for grinder/furnace: takes 33* as long and consumes twice the power, but processes whole stack at once.

Or upgrade for quarry:  
takes one minute per layer, instead of digging one node each time.

SX
https://github.com/mt-mods/technic/issues/171 and https://github.com/mt-mods/technic/issues/108

Oh, also this might make really big difference https://github.com/mt-mods/technic/issues/148 
but would also mean possibly losing stuff if server crashes.

About making machines more expensive, that method cannot really be used to reduce machines actually used in world. Cost simply cannot be increased and it would only make machines only craftable by players who already have a lot of resources and for those players there's no real limits. It would be unfair and not exactly useful.

Basically there's no really expensive thing unless it requires materials that absolutely cannot be automated in any way.

Currently I think only somewhat expensive things are xenomorph eggs, oranges, bananas and strawberries. Somewhat expensive things also include ore nodes (nodes not lumps or ores). Those can be used in recipes for some expensive things but does not really fit machine recipes that well.

**Comment by Klaranth** [comment]
Huhhila
Could those slower-cycle grinders/furnaces be 2x2x2=8 or 3x3x3=27 nodes in size, or are there still bunch of bugs related to items bigger than 1 node (JD slicing and beyond)? How many ways can bigger-than-1-node items be implemented anyway?

Shadow
What are some examples of multi-node items?  the travelnet box, what else?

Huhhila
Some bridgers are like 5x2x1. And some people seem to be fans of "vendor glass", which relies on a somewhat similarly odd mechanism as rotated "heavy/silent" water does...

**Comment by S-S-X** [comment]
> Huhhila
> Could those slower-cycle grinders/furnaces be 2x2x2=8 or 3x3x3=27 nodes in size, or are there still bunch of bugs related to items bigger than 1 node (JD slicing and beyond)? How many ways can bigger-than-1-node items be implemented anyway?

Machines can be larger and require actual nodes instead of being multi node items. That could possibly allow somewhat balanced speed upgrades for machines.

Basically problem with simple updates that just make machines faster is that this continues forever, players who do have way more than they can use are demanding faster processing of things so that they can have even more things that they cannot really use. It is more about feeling wealth and power than actually requiring speed upgrades for anything.

Another problem with such upgrades is that those make lower tier machines obsolete and useless, on servers where there is public workshops available this effect can be immediate and only top tier machines will be used. On servers without public workshops low tier machines are temporary step you want to get past as soon as possible.

Having such obsolete things in game is not useful especially if they are not exactly require at any point in game, things like that are just consuming server resources without providing anything useful.
As for upgrades, if those make machines way better than what they are without upgrades situation gets very similar: you wont use machines without over powered upgrades anymore anywhere.

Also there's clear pattern in these requests, one issue asks for more powerful machines that are faster while another asks for upgraded generators that will provide more power. Basically asking to just increase game speed so that everything happens faster.

Upgrades like this will require a lot more careful planning to actually provide content that can work together with already available things.

**Comment by S-S-X** [comment]
Linking few issues that mostly aim to include shortcuts for speeding up game, reducing required space for machines, making mining or processing simply faster as main goal without providing much anything else: #164 #159 #136 #118 #114 #106 #81
